packages/main/src/components/Form/index.tsx

```diff
@@ -101,6 +101,7 @@ export interface FormPropTypes extends CommonProps {
 
 const clonedChildrenForSingleColumn = (reactChildren, currentLabelSpan) =>
   React.Children.map(reactChildren, (child) => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
`@ts-ignore`, shouldn't be necessary here, I probably overlooked that in the initial implementation

Reply:
done